source/extensions/filters/udp/udp_proxy/udp_proxy_filter.h

```diff
@@ -119,6 +119,10 @@ class UdpProxyFilter : public Network::UdpListenerReadFilter,
     const Upstream::Host& host() const { return *host_; }
     void write(const Buffer::Instance& buffer);
 
+    // Network::UdpPacketProcessor
+    void disable() override { socket_event_->setEnabled(0); }
```

Comment:
I cannot add more test for the UdpProxyFilter::ActiveSession because it is a private class so that the gtest cannot access this class.
So, the code coverage for source/extensions/filters/udp is lower than limit but I cannot add more tests. :(

> Code coverage for extension source/extensions/filters/udp is lower than limit of 91.1 (91.0)